huaweicloud/resource_huaweicloud_bcs_instance.go

```diff
@@ -0,0 +1,760 @@
+						},
+					},
+				},
+			},
+			"tc3_need": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"restful_api_support": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"delete_obs": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"delete_storage": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"delete_resource": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"version": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"purchase_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"cross_region_support": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"rollback_support": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"old_service_version": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"agent_portal_address": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+		},
+	}
+}
+
+func resourceBCSInstanceV2Create(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*config.Config)
+	client, err := config.BcsV2Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("Error creating HuaweiCloud Blockchain client: %s", err)
+	}
+	isCreateNewCluster := false
+	createOpts := blockchains.CreateOpts{
+		Name:                d.Get("name").(string),
+		VersionType:         d.Get("version_type").(int),
+		FabricVersion:       d.Get("fabric_version").(string),
+		BlockChainType:      d.Get("blockchain_type").(string),
+		Consensus:           d.Get("consensus").(string),
+		SignAlgorithm:       d.Get("sign_algorithm").(string),
+		VolumeType:          d.Get("volume_type").(string),
+		EvsDiskType:         d.Get("evs_disk_type").(string),
+		OrgDiskSize:         d.Get("org_disk_size").(int),
+		DatabaseType:        d.Get("database_type").(string),
+		Password:            d.Get("password").(string),
+		OrdererNodeNumber:   d.Get("orderer_node_num").(int),
+		EIPEnable:           d.Get("eip_enable").(bool),
+		BandwidthSize:       d.Get("bandwidth_size").(int),
+		ClusterType:         d.Get("cluster_type").(string),
+		TC3Need:             d.Get("tc3_need").(bool),
+		RestfulAPISupport:   d.Get("restful_api_support").(bool),
+		CreateNewCluster:    &isCreateNewCluster,
+		EnterpriseProjectId: GetEnterpriseProjectID(d, config),
+		CCEClusterInfo:      resourceBCSClusterInfo(d),
+		PeerOrgs:            resourceOrgPeer(d),
+		Channels:            resourceChannelInfo(d),
+		CouchDBInfo:         resourceCouchDBInfo(d),
+		SFSTurbo:            resourceTurboInfo(d),
+		Block:               resourceBlockInfo(d),
+		Kafka:               resourceKafkaCreateInfo(d),
+	}
+	log.Printf("[DEBUG] Create Blockchain Options: %#v", createOpts)
```

Comment:
the log may include `password` field which is security.

Reply:
Thanks, that's not secure.